### PR TITLE
ACS-3227 Create folder rules.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java [hunk 1]
```diff
@@ -288,4 +292,21 @@ public static String keyValueJson(HashMap<String, String> mapJson)
...
         return builder.build().toString();
     }
+
+    /**
+     * Convert a collection of {@link RestRuleModel} objects to JSON for a multi-entity POST request.
+     *
+     * @param models The entities to convert.
+     * @return The JSON string.
+     */
+    public static String arrayToJson(List<RestRuleModel> models)
```
Comment (tpage-alfresco) [inline src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java]: This shouldn't mention rules.

Comment (tpage-alfresco) [reply src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java]: Fixed

**Review by gevagel — APPROVED** [review]
LGTM

File: src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java [hunk 2]
```diff
@@ -1,13 +1,14 @@
 package org.alfresco.rest.core;
 
-import java.util.HashMap;
```
Comment (montgolfiere) [inline src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java:3]: Minor: missing lic headers ?

I realise not specific to this PR. To get the ball rolling, ,maybe add as we touch files (pending a future sweep) ?

**Review by montgolfiere — APPROVED** [review]

File: src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java (hunk 2 above)
Comment (tpage-alfresco) [reply src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java:3]: After a quick survey of the project then I didn't spot any classes with license headers except the FolderRules class created by Kris.  I'll work on a second PR to add the license header plugin to this project.

**Review by KikoUK — APPROVED** [review]

**Pull request merged**